core/src/components/input/input.tsx

```diff
@@ -41,6 +41,8 @@ export class Input implements ComponentInterface {
 
   /**
    * Indicates whether and how the text value should be automatically capitalized as it is entered/edited by the user.
+   * Default options are: `"off"`, `"characters"`, `"words"`, and `"sentences"`.
+   * For more information, see [Autocapitalize for mobile](https://developers.google.com/web/updates/2015/04/autocapitalize)
    */
   @Prop() autocapitalize = 'off';
```

Comment:
Rather than explicitly list the values in the documentation, we can make it a type here instead. The docs will automatically pull the values in, and users will also get added TypeScript checking too.

```suggestion
  @Prop() autocapitalize: AutocapitalizeTypes = 'off';
```

You would add the type definition here: https://github.com/ionic-team/ionic-framework/blob/master/core/src/interface.d.ts#L40

```
export type AutocapitalizeTypes = 'off' | 'none' | 'on' | 'sentences' | 'words' | 'characters';
```

Reply:
For an example of how this looks, check out `inputmode` or `enterkeyhint` in the ion-input properties: https://ionicframework.com/docs/api/input#properties

Reply:
@liamdebeasi Thanks for pointing that out